composer.json

```diff
@@ -12,41 +12,30 @@
         "php": ">=7.0",
         "ext-pcntl": "*",
         "guzzlehttp/guzzle": "^6.0",
-        "smartbox/besimple-soap": "^1.2",
-        "smartbox/core-bundle": "~1.4",
-        "symfony/expression-language": "^2.8",
-        "symfony/framework-bundle": "^2.8",
-        "symfony/options-resolver": "^2.8",
-        "symfony/validator": "^2.8"
+        "smartbox/besimple-soap": "^1.3",
+        "smartbox/core-bundle": "~1.6",
+        "symfony/expression-language": "^2.8 || ^3.4",
+        "symfony/framework-bundle": "^2.8.18 || ^3.4",
```

Comment:
\>=2.8.18 <2.0.0: https://getcomposer.org/doc/articles/versions.md#caret-version-range-

> The ^ operator behaves very similarly but it sticks closer to semantic versioning, and will always allow non-breaking updates. For example ^1.2.3 is equivalent to >=1.2.3 <2.0.0 as none of the releases until 2.0 should break backwards compatibility. 

The reason I used 2.8.18 is because previous versions use older versions of PHPUnit and breaks our tests (we use TestCase and PHPUnit < 6.5 uses PHPUnit_Test_Case)